src/main/java/MockClass.java

```diff
@@ -5,4 +5,27 @@ public int doSomething(int seed) {
 
         return seed + 5;
     }
+
+    public static void main(String[] args) {
+
+        Day day = null;
+
+        switch (day) {
+            case MONDAY:
+            case TUESDAY:
+                WEDNESDAY:   // Noncompliant; syntactically correct, but behavior is not what's expected
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this misleading "WEDNESDAY" label. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3AS1219)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor the code to remove this label and the need for it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ALabelsShouldNotBeUsedCheck)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused label. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3AS1065)